src/everest/util/__init__.py

```diff
@@ -46,11 +37,6 @@ def configure_logger(
         handler.setFormatter(logging.Formatter(formatter))
         logger.addHandler(handler)
 
-    # Setup azure logging if needed
-    azure_handler = get_azure_logging_handler()
```

Comment:
We should just follow the same pattern as ert:
```python
    def add_logging_handle_to_root(self, logger: logging.Logger) -> None:
        handles = self.hook.add_log_handle_to_root()
        for handle in handles:
            logger.addHandler(handle)
```

Reply:
https://github.com/equinor/ert/blob/af3600938b9d49683035a1d32eed43af53131cf7/src/ert/plugins/plugin_manager.py#L374